src/containers/my-resources/attachments-container/AttachmentsContainer.tsx

```diff
@@ -82,72 +65,51 @@ const AttachmentsContainer = () => {
     []
   )
 
-  const updateAttachment = useCallback(
-    (params?: UpdateAttachmentParams) =>
-      ResourceService.updateAttachment(params),
-    []
-  )
-
   const {
-    response,
-    loading,
-    fetchData: fetchAttachments
-  } = useAxios<ItemsWithCount<Attachment>, GetResourcesParams>({
-    service: getAttachments,
-    defaultResponse: defaultResponses.itemsWithCount,
-    onResponseError
+    data: loadedAttachments,
+    isLoading: isLoadingAttachments,
+    refetch: refetchAttachments,
+    error: attachmentsLoadError
+  } = useQuery<ItemsWithCount<Attachment>>({
```

Comment:
You can remove those types since we don't have requirements for mandatory typing in our project. `useQuery` will be typed by `baseService` and `queryFn` itself
```suggestion
  } = useQuery({
```